packages/elements/src/clock/index.ts

```diff
@@ -220,27 +241,49 @@ export class Clock extends ResponsiveElement {
   /**
    * Display the digital clock in 12hr format.
    */
-  @property({ type: Boolean, attribute: 'am-pm' })
+  @property({ type: Boolean, attribute: 'am-pm', reflect: true })
   public amPm = false;
 
   /**
    * Display the seconds segment.
    */
-  @property({ type: Boolean, attribute: 'show-seconds' })
+  @property({ type: Boolean, attribute: 'show-seconds', reflect: true })
```

Comment:
Why reflect ?

Reply:
It actually is an idea from Alek. It should be reflect because it effects to UI.

Reply:
to be inline with the time-picker. Potentially these properties can have UI differences.

Reply:
Got it. Thanks.